doc/conf.py

```diff
@@ -0,0 +1,77 @@
+import pathlib
+import warnings
+
+import packaging.version
+
+
+needs_sphinx = '4.0'
+
+project = 'qutip-tensorflow'
+author = 'QuTiP developers'
+copyright = '2021 and later, ' + author
+
+
+def _check_imported_local_package():
+    """
+    Warn if the importable version of the package is not in the same repository
+    as this documentation.  The imported version is expected to have been made
+    available by ::
+
+        pip install -e .
+
+    or similar; if someone is trying to use a release version to build the docs
+    from a local checkout, the versions likely will not match and they'll end
+    up with a chimera.
+    """
+    import qutip_tensorflow
+    repo_dir = pathlib.Path(__file__).absolute().parents[1]
+    expected_import_dir = repo_dir / 'src' / 'qutip_tensorflow'
+    imported_dir = pathlib.Path(qutip_tensorflow.__file__).parent
+    if expected_import_dir != imported_dir:
+        warnings.warn(
+            "The version of qutip_tensorflow available on the path is not "
+            "from the same location as the documentation.  This may result in "
+            "the documentation containing text from different sources."
+            f"\nDocumentation source   : {str(repo_dir)}"
+            f"\nImported package source: {str(imported_dir)}"
+        )
+
+
+def _version():
+    _check_imported_local_package()
+    filename = pathlib.Path(__file__).absolute().parents[1] / 'VERSION'
+    with open(filename, "r") as file:
+        version = file.read().strip()
+    # Canonicalise the version format, just in case.
+    return str(packaging.version.parse(version))
+
+
+release = _version()
+
+extensions = [
+    'sphinx.ext.mathjax',
+    'sphinx.ext.napoleon',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.autodoc',
+]
+
+# Patterns to exclude when looking for sources in the build.
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+
+
+# HTML setup
+
+html_theme = 'sphinx_rtd_theme'
+# Directories for static files to be copied across after the build.
+html_static_path = []
```

Comment:
Yeah, any directories in `html_static_path` get copied across verbatim, which lets you override things in the theme's style files, or manage your own website assets.

The reason I didn't put anything in here at first is because we don't have any additional assets, so there's no need to define what you'll call the folder you put them in.  If I had, it can cause slightly problems because of how `git` works.  I could specify a directory here, but if it doesn't exist, Sphinx may throw an error (I'm not actually certain if it does any more).  However, `git` doesn't track directories, only files, so I can't automatically create the repository on `git clone` unless I put a dummy file into it.  I could also add a dummy target into the Makefile to ensure the directory is always created, but again, that's quite messy.

Seems better to me just to leave it empty unless you decide you want additional static assets.  It doesn't matter if it's consistent with the main repository or not, really - there's a lot of things we'd like to improve in the main repo, so we shouldn't feel too constrained with things like the documentation.